deeplearning4j-scaleout/pom.xml

```diff
@@ -37,6 +37,12 @@
         <module>deeplearning4j-scaleout-parallelwrapper-parameter-server</module>
     </modules>
 
+    <properties>
+        <spark.major.version>1</spark.major.version>
+        <spark.version>1.6.2</spark.version>
+        <datavec.spark.version>0.7.3_spark_${spark.major.version}-SNAPSHOT</datavec.spark.version>
```

Comment:
These can't be defined in scaleout: there's only one scaleout pom, so it doesn't solve the problem.
they need to be in deeplearning4j-scaleout/spark/pom.xml, and then separately again in deeplearning4j-scaleout/dl4j-streaming/pom.xml